JSDemos/Demos/Form/CustomizeItem/AngularJS/styles.css

```diff
@@ -1,11 +1,3 @@
 #form-container {
   margin: 10px 10px 30px;
 }
-
```

Comment:
I think we should update code for AngularJS while this demo is visible on our site

Reply:
https://teams.microsoft.com/l/message/19:7a8fe7395a2d47288c29cfc421df6cbf@thread.skype/1665578915607?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=8c31b3e5-5870-46b1-92a7-180d13b282ed&parentMessageId=1662721259902&teamName=DevExtreme&channelName=General&createdTime=1665578915607&allowXTenantAccess=false